### ISSUE TITLE
Grid find field missing label

### ISSUE THREAD

**Issue by Blackbaud-MattGregg** [blackbaud/skyux1#211]
Grid toolbar find in field missing label.
Add label "Find in this list" and associate to the find input field. 

Technique - 
Add before input field and "hide" offscreen w/ CSS.
Associate label to input field with for and id attributes.

**Comment by Blackbaud-PatrickOFriel** [comment]
I believe this has been solved by adding an `aria-label` when using the new search component in `bb-grid`
